components/network/samba/Makefile

```diff
@@ -160,7 +168,7 @@ REQUIRED_PACKAGES += library/print/cups-libs
 REQUIRED_PACKAGES += library/readline
```

Comment:
library/icu is also needed (it's used if latest version is found on build host).